integration-tests/api/__tests__/admin/product.js

```diff
@@ -27,52 +32,16 @@ let {
 
 jest.setTimeout(50000)
 
-const productFixture = {
```

Comment:
@sradevski, had to move the fixture inside the `moduleIntegrationTestRunner`. When passing the environment variable directly to the runner instead of setting it in `.env`, it is not globally available.

Reply:
Sounds good 👍